vizro-ai/hatch.toml

```diff
@@ -35,6 +35,7 @@ prep-release = [
   "hatch version release",
   "hatch run changelog:scriv collect --add",
   "hatch run changelog:add",
+  "hatch run lint || hatch run lint",
```

Comment:
I'll fix this later today after I check what happened with the vizro-core process.

Reply:
This is how we should handle it: #165. Exactly the same code here in hatch.toml should work.